docs/config/index.md

```diff
@@ -366,18 +366,37 @@ export default defineConfig({
 })
 ```
 
-To get TypeScript working with the global APIs, add `vitest/globals` to the `types` field in your `tsconfig.json`
+#### TypeScript integration
 
-```json
-// tsconfig.json
-{
-  "compilerOptions": {
-    "types": ["vitest/globals"]
-  }
-}
-```
+To ensure TypeScript recognizes the global APIs, you have two options:
+
+1. Import the global types
+
+    Do a type import from `vitest/globals` in any `.ts` file (e.g., your [setup file](#setupfiles)). This registers the global APIs types automatically. Ensure the file is included in your `tsconfig.json` so TypeScript knows it's apart of the same project.
+
+    ```ts
+    // test-setup.ts
+    import type {} from 'vitest/globals'
```

Comment:
If this can be updated to also include an empty file for `default`:
https://github.com/vitest-dev/vitest/blob/main/packages/vitest/package.json#L37-L39

We can simplify the recommendation to `import 'vitest/globals'`